GOVERNANCE.rst

```diff
@@ -0,0 +1,140 @@
+.. _governance:
+
+WarpX Governance
+================
+
+WarpX is led in an open governance model, described in this file.
+
+
+Steering Committee
+------------------
+
+Current Roster
+^^^^^^^^^^^^^^
+
+- Jean-Luc Vay (chair)
+- Remi Lehe
+- Axel Huebl
+
+See: `GitHub team <https://github.com/orgs/ECP-WarpX/teams/warpx-admins>`__
+
+Role
+^^^^
+
+The steering committee (SC) can change organizational settings, do administrative operations such as rename/move/archive repositories, change branch protection rules, etc.
```

Comment:
Clarified per discussion:
```suggestion
Members of the steering committee (SC) can change organizational settings, do administrative operations such as rename/move/archive repositories, change branch protection rules, etc.
```
SC members can change branch protection rules, etc. as needed for daily administrative work (e.g., CI).